enos/modules/aws_bucket/variables.tf

```diff
@@ -12,7 +12,14 @@ variable "cluster_tag" {
   type        = string
 }
 
+variable "is_user" {
+  description = "Boolean to specify if a user was provided to this module."
+  type        = bool
+  default     = false
+}
+
```

Comment:
I'm not super happy about this implementation, but after discussing with others, there doesn't look to be an alternative at the moment.

I originally only had the modification to the `user` variable and added the check...
```
count = var.user != "" ? 1 : 0
```
but Terraform was not happy with that because it couldn't determine the value of `var.user` during the plan.